_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/styles/_articles.scss

```diff
@@ -70,5 +70,13 @@
 @include desktop-and-down {
   #rhd-article {
     padding: 15px;
-  }  
+  }
+}
+
+.page-node-type-article .disqus-thread-wrapper {
```

Comment:
Are we only displaying Disqus on articles?

Reply:
No, there's some styles in other partials that have the disqus thread. The pages are all a little different and required some varying padding and whatnot to match the pages they were on. (Should be articles, books, cheat sheets, and video resources.)

Reply:
That's less ideal (all the places having slightly different styles), but probably can't be easily remedied here.